unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Controllers/BIWEntityHandler.cs

```diff
@@ -272,23 +274,23 @@ public override void ExitEditMode()
         base.ExitEditMode();
         DeselectEntities();
 
-        foreach (DCLBuilderInWorldEntity entity in convertedEntities.Values)
+        foreach (BIWEntity entity in convertedEntities.Values)
         {
             entity.Dispose();
         }
 
         convertedEntities.Clear();
     }
 
-    private void ChangeEntitySelectionFromList(DCLBuilderInWorldEntity entityToEdit)
+    private void ChangeEntitySelectionFromList(BIWEntity entityToEdit)
     {
         if (!selectedEntities.Contains(entityToEdit))
             SelectFromList(entityToEdit);
         else
             DeselectEntity(entityToEdit);
```

Comment:
If the opposite of `SelectFromList` is `DeselectEntity` I think we should pair the names. 

Some suggestions:
```
SelectEntity(entity)
DeselectEntity(entity)
```
```
SelectFromList(entity)
DeselectFromList(entity)
```

Reply:
Is not the opposite, right now there is no difference between deselect and entity, but there is a difference when selecting entities, `SelectFromList `is not the same as `SelectEntity `. So in this case, we can't pair the names